build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.5.5"
+    id("io.spring.dependency-management") version "1.0.11.RELEASE"
```

Comment:
To me, it seems that you are not using this plugin. Could you please elaborate on why do you need it?

Reply:
It's also generated by Spring for future development. Removed in 5699e74